### PR TITLE
Add notes from CPC Meeting on 2020-01-14

### LINKED ISSUE #444: OpenJS Foundation Cross Project Council Meeting 2020-01-14
## Time

**UTC Tue 14-Jan-2020 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 14-Jan-2020 09:00 (09:00 AM) |
| US / Mountain | Tue 14-Jan-2020 10:00 (10:00 AM) |
| US / Central  | Tue 14-Jan-2020 11:00 (11:00 AM) |
| US / Eastern  | Tue 14-Jan-2020 12:00 (12:00 PM) |
| London        | Tue 14-Jan-2020 17:00 (05:00 PM) |
| Amsterdam     | Tue 14-Jan-2020 18:00 (06:00 PM) |
| Moscow        | Tue 14-Jan-2020 20:00 (08:00 PM) |
| Chennai       | Tue 14-Jan-2020 22:30 (10:30 PM) |
| Hangzhou      | Wed 15-Jan-2020 01:00 (01:00 AM) |
| Tokyo         | Wed 15-Jan-2020 02:00 (02:00 AM) |
| Sydney        | Wed 15-Jan-2020 04:00 (04:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Cross%20Project%20Council+Meeting+2020-01-14&iso=20200114T17
* or http://www.wolframalpha.com/input/?i=05PM+UTC%2C+Jan+14%2C+2020+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1ujxs2ZIjM_hbdIhd9WjuNS2EZISj3TbiFKIL-Qj6wpk/edit>

## Agenda

Extracted from **cross-project-council-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/cross-project-council

* Start a Code of Conduct Working Group [#432](https://github.com/openjs-foundation/cross-project-council/issues/432)
* Add Foundation-wide copyright guidance [#414](https://github.com/openjs-foundation/cross-project-council/pull/414)
* Kicking off an OpenJS Foundation project landscape [#411](https://github.com/openjs-foundation/cross-project-council/issues/411)
* adds proposal for charter submission & review per #200 [#405](https://github.com/openjs-foundation/cross-project-council/pull/405)
* Add post-graduation checklist [#386](https://github.com/openjs-foundation/cross-project-council/pull/386)
* doc: move COC proposal to stage 3 [#379](https://github.com/openjs-foundation/cross-project-council/pull/379)
* Clarify Travel Fund status & process [#377](https://github.com/openjs-foundation/cross-project-council/issues/377)
* Responsible security disclosures [#326](https://github.com/openjs-foundation/cross-project-council/issues/326)
* Post-Bootstrap work to be managed by CPC [#115](https://github.com/openjs-foundation/cross-project-council/issues/115)

### openjs-foundation/standards

* OSI Affiliate Membership [#32](https://github.com/openjs-foundation/standards/issues/32)

### openjs-foundation/project-onboarding

* Communication channel between CPC and AMP infra WG [#31](https://github.com/openjs-foundation/project-onboarding/issues/31)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `cross-project-council-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: https://zoom.us/j/920010234
* For those who just want to watch: https://livestream.openjsf.org

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by obensource**
## Description
This PR publishes the meeting notes from the CPC's meeting on 2020-01-14.

Closes #444.

🎉

### CODE REVIEW

**Comment by joesepi** [comment]
I’ll request this to be fast tracked. Need two thumbs from members on this comment to proceed.

**Comment by MylesBorins** [comment]
@joesepi I'm not 100% that notes are something that should be fast tracked. It is good to give folks a chance to review and ensure they are accurate... it doesn't really hurt anyone keeping it open a bit longer

thoughts?

File: meetings/2020-01-14.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+* Anton Molleda (@molant)
+* Jory Burson (@jorydotcom)
+* Timmy Willison (@timmywil)
+* Jordan Harband (@LJHarb)
+* Christian Bromann (@christian-bromann)
+* Sendil Kumar (@sendilkumarn)
+* Ben Michel (@obensource)
+* Tierney Cyren (@bnb)
+* Mike Samuel (@mvsamuel)
+
```
Comment (codeekage) [inline meetings/2020-01-14.md]: I joined but dropped 30 mins into the meeting :)

**Comment by joesepi** [comment]
Yeah @MylesBorins -- no need to fast track minutes. I do it just to keep things tidy.

File: meetings/2020-01-14.md (hunk 1 above)
Comment (joesepi) [reply meetings/2020-01-14.md]: You were there though! No change needed here.

**Pull request merged**